tests/console/command_not_found.pm

```diff
@@ -33,34 +33,25 @@ sub run {
     my $not_installed_pkg = (is_sle && sle_version_at_least '15') ? 'wireshark' : 'xosview';
     my $cnf_cmd = qq{echo "\$(cnf $not_installed_pkg 2>&1 | tee /dev/stderr)" | grep -q "zypper install $not_installed_pkg"};
 
-    my $result = script_run($cnf_cmd);
-
     save_screenshot;
+    # Return if command execution was successful
+    return unless script_run($cnf_cmd);
 
-    if ($result) {
-        if (is_sle && sle_version_at_least '15') {
-            $self->{run_post_hook} = 1;
-            record_soft_failure 'https://fate.suse.com/323424';
-            select_console 'root-console';
-            add_suseconnect_product("sle-module-desktop-applications");
-            zypper_call("ref");
-            select_console 'user-console';
-        }
-        else {
-            die "Command Not Found failed";
-        }
+    # Soft-fail if command execution fails on sle 15
+    if (is_sle && sle_version_at_least '15') {
+        record_soft_failure 'https://fate.suse.com/323424';
+        select_console 'root-console';
+        script_run('SUSEConnect -s');
+        add_suseconnect_product("sle-module-desktop-applications");
+        zypper_call("ref");
```

Comment:
to be consistent would you mind to use single ticks by default everywhere?

Reply:
Absolutely